src/hotspot/share/utilities/numberSeq.cpp

```diff
@@ -22,7 +22,7 @@
  *
  */
 
-#include "memory/allocation.inline.hpp"
+#include "memory/allocation.hpp"
```

Comment:
My rule is if there is a `.inline.hpp` file, always include it instead of the `.hpp` file in the `.cpp` to be sure to catch everything (the inline.hpp always includes the .hpp); and the allocation.hpp must already be included in `numberSeq.hpp`.

This just gives headaches imo when more code is moved to the `.inline.hpp` file. It works this time because nothing from `CHeapObj` is in the `inline.hpp` file, so technically the removal of this include entirely works (as mentioned in the other comment, `numberSeq.hpp` needs to include `allocation.hpp`).